.github/workflows/build_executables.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Sweyer executables
+on: 
+  workflow_dispatch
```

Comment:
Let's make it run on each PR and make it required

If some PR breaks app build, it should not be merged

Reply:
Ok, done👍